xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogCode.xml

```diff
@@ -31,8 +30,8 @@
   <parent>Blog.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <date>1373018317000</date>
-  <contentUpdateDate>1373018155000</contentUpdateDate>
+  <date>1477383233000</date>
+  <contentUpdateDate>1477383233000</contentUpdateDate>
```

Comment:
I export  a xar and launch a xar:format instead to manualy edit the file but I get it.